source/common/upstream/upstream_impl.h

```diff
@@ -453,6 +455,7 @@ class ClusterInfoImpl : public ClusterInfo {
   const bool added_via_api_;
   LoadBalancerSubsetInfoImpl lb_subset_;
   const envoy::api::v2::core::Metadata metadata_;
+  ::Envoy::Config::TypedMetadataImpl<ClusterTypedMetadataFactory> typed_metadata_;
```

Comment:
Nit: Use relative namespace paths as elsewhere in this file.

Reply:
done